public/service/yaml.js

```diff
@@ -83,12 +80,10 @@ var configTx = function(json) {
   jsonyaml.Organizations = orgs;
   let yaml = yamljs.stringify(jsonyaml);
 
-  // format label...
-
-  let index = yaml.indexOf('&') + json.name.length;
-  yaml = yaml.substr(0, index) + yaml.substr(index + 2);
+  // remove the ':' from the org anchor
+  yaml = yaml.replace(`${json.name}:`, `${json.name}`);
```

Comment:
This is the bug fix for the stripping of the single character off the org anchor in the yaml file.